addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -370,6 +370,57 @@ options.Class.include({
             },
         });
     },
+    /**
+     * @private
+     * @param {String} type either 'head' or 'footer'
+     */
+    async openCustomCodeDialog(previewMode, widgetValue, params) {
+        let websiteId;
+        this.trigger_up('context_get', {
+            callback: (ctx) => {
+                websiteId = ctx['website_id'];
+            },
+        });
+        const websites = await this._rpc({
+            model: 'website',
+            method: 'read',
+            args: [[websiteId], ['custom_code_head', 'custom_code_footer']],
+        });
+
+        const $content = $(core.qweb.render('website.custom_code_dialog_content', {
+            text: _.str.sprintf(_t('Enter code that will be added into the "%s" tag of every page of your site.'), widgetValue),
+        }));
+
+        const dialog = new Dialog(this, {
+            title: widgetValue === 'footer' ? 'Footer' : 'Head',
```

Comment:
not translatable ?

Reply:
I hesitated to tell too... but we are in fact talking about `<head/>` and `<footer/>` so I am not sure about the actual words to use here. @sbuhl ?

Reply:
@qsm-odoo 
The use of the term "footer" is to vulgarize the fact that we add the code just before </body>.
Hence, "head" seems technically correct for me but footer is questionable. 
(in fact @ThanhDodeurOdoo no need to proper capitalize, you were right, sorry).

So, not sure it needs to be translated.